### PR TITLE
OSDOCS-10716: updates audit log default values

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.16+

Issue:
[OSDOCS-10716](https://issues.redhat.com/browse/OSDOCS-10716)

Link to docs preview:
[Configuring audit logs](https://76743--ocpdocs-pr.netlify.app/microshift/latest/microshift_configuring/microshift-audit-logs-config.html)

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

### CODE REVIEW

**Comment by copejon** [comment]
Thanks @ShaunaDiaz, the audit log changes LGTM

**Comment by ShaunaDiaz** [comment]
QE approval given on Slack, awaiting LGTM on PR--OK for peer review.

File: modules/microshift-audit-logs-config-proc.adoc [hunk 1]
```diff
@@ -20,15 +20,15 @@ You can configure audit log settings by using the {microshift-short} service con
...
+<1> Specifies the maximum time in days that log files are kept. Files older than this limit are deleted. If the value is 0, the limit is disabled. In this example, after a log file is more than 7 days old, it is deleted. The files are deleted regardless of whether or not the live log has reached the maximum file size specified in the `maxFileSize` field. File age is determined by the timestamp written in the name of the rotated log file, for example, `audit-2024-05-16T17-03-59.994.log`.
+<2> The maximum audit log file size in megabytes. If the value is 0, the limit is disabled. In this example, the file is rotated as soon as the live log reaches the 200 MB limit.
+<3> The maximum number of rotated audit log files retained. After the limit is reached, the log files in order from oldest to newest are deleted. When the value is 0, the limit is disabled. In this example, the value `1` results in only 1 file of size `maxFileSize` being retained in addition to the current active log.
```
Comment (mburke5678) [inline modules/microshift-audit-logs-config-proc.adoc:31]:
```suggestion
<3> The maximum number of rotated audit log files retained. After the limit is reached, the log files are deleted in order from oldest to newest. When the value is `0`, the limit is disabled. In this example, the value `1` results in only 1 file of size `maxFileSize` being retained in addition to the current active log.
```

File: modules/microshift-audit-logs-config-proc.adoc [hunk 2]
```diff
@@ -20,15 +20,15 @@ You can configure audit log settings by using the {microshift-short} service con
...
-<3> Specifies the maximum time in days that log files are kept. Files older than this limit are deleted. If the value is 0, the limit is disabled. In this example, after a log file is more than 7 days old, it is deleted. The files are deleted regardless of whether or not the live log has reached the maximum file size specified in the `maxFileSize` field. File age is determined by the timestamp written in the name of the rotated log file, for example, `audit-2024-05-16T17-03-59.994.log`.
+<1> Specifies the maximum time in days that log files are kept. Files older than this limit are deleted. If the value is 0, the limit is disabled. In this example, after a log file is more than 7 days old, it is deleted. The files are deleted regardless of whether or not the live log has reached the maximum file size specified in the `maxFileSize` field. File age is determined by the timestamp written in the name of the rotated log file, for example, `audit-2024-05-16T17-03-59.994.log`.
```
Comment (mburke5678) [inline modules/microshift-audit-logs-config-proc.adoc:29]: s/If the value is 0,/If the value is `0`,
s/ after a log file/when a log file

File: modules/microshift-audit-logs-config-proc.adoc [hunk 3]
```diff
@@ -20,15 +20,15 @@ You can configure audit log settings by using the {microshift-short} service con
...
+<2> The maximum audit log file size in megabytes. If the value is 0, the limit is disabled. In this example, the file is rotated as soon as the live log reaches the 200 MB limit.
```
Comment (mburke5678) [inline modules/microshift-audit-logs-config-proc.adoc:30]:
```suggestion
<2> The maximum audit log file size in megabytes. If the value is `0`, the limit is disabled. In this example, the file is rotated as soon as the live log reaches the 200 MB limit.
```

File: modules/microshift-audit-logs-config-intro.adoc [hunk 4]
```diff
@@ -20,11 +20,18 @@ You can set fields in combination to define a maximum storage limit for retained
...
-* `maxFiles`: 10 files
-* `maxFileAge`: 0, This value means that no default age limit is set.
-* `profile`: Default, This profile logs only metadata for read and write requests.
+.{microshift-short} default audit log values
+[cols="20%,20%,50%",options="header"]
+|===
 
-The maximum default storage usage for audit log retention is 2000Mb, provided that there are 10 or fewer files.
+|Audit log parameter|Default setting|Definition
+|`maxFileAge`:|`0`|How long log files are kept before automatic deletion. The default value means that a log file is never deleted based on age. This value can be configured.
```
Comment (mburke5678) [inline modules/microshift-audit-logs-config-intro.adoc:28]:
```suggestion
|`maxFileAge`:|`0`|How long log files are kept before automatic deletion. The default is `0`, which means that a log file is never deleted based on age. This value can be configured.
```

File: modules/microshift-audit-logs-config-intro.adoc [hunk 5]
```diff
@@ -20,11 +20,18 @@ You can set fields in combination to define a maximum storage limit for retained
...
+|`maxFiles`:|`10`|The total number of log files kept. By default, {microshift-short} retains 10 log files. The oldest is deleted when an excess file is created. This value can be configured.
```
Comment (mburke5678) [inline modules/microshift-audit-logs-config-intro.adoc:29]: I personally like _retained_. To me it sounds better and it matches the following sentence.

```suggestion
|`maxFiles`:|`10`|The total number of log files retained. By default, {microshift-short} retains 10 log files. The oldest is deleted when an excess file is created. This value can be configured.
```

File: modules/microshift-audit-logs-config-intro.adoc [hunk 6]
```diff
@@ -20,11 +20,18 @@ You can set fields in combination to define a maximum storage limit for retained
...
+|`maxFileSize`:|`200`|In megabytes. By default, when the `audit.log` file reaches the `maxFileSize` limit, the `audit.log` file is rotated and {microshift-short} begins writing to a new `audit.log` file. This value can be configured.
```
Comment (mburke5678) [inline modules/microshift-audit-logs-config-intro.adoc:30]:
```suggestion
|`maxFileSize`:|`200`|By default, when the `audit.log` file reaches the `maxFileSize` limit, the `audit.log` file is rotated and {microshift-short} begins writing to a new `audit.log` file. This value is in megabytes and can be configured.
```

File: modules/microshift-audit-logs-config-intro.adoc [hunk 7]
```diff
@@ -20,11 +20,18 @@ You can set fields in combination to define a maximum storage limit for retained
...
+|`profile`:|`Default`|Logs only metadata for read and write requests; does not log request bodies except for OAuth access token requests. If you do not specify this field, the `Default` profile is used.
```
Comment (mburke5678) [inline modules/microshift-audit-logs-config-intro.adoc:31]: The word _Logs_ here confused me; at first I thought it was a noun.

```suggestion
|`profile`:|`Default`|Only metadata is logged for read and write requests; does not log request bodies except for OAuth access token requests. If you do not specify this field, the `Default` profile is used.
```

Are there other profiles that can be used?

**Comment by mburke5678** [comment]
@ShaunaDiaz A few comments. Otherwise LGTM.

File: modules/microshift-audit-logs-config-intro.adoc (hunk 5 above)
Comment (ShaunaDiaz) [reply modules/microshift-audit-logs-config-intro.adoc:29]: The best word is probably "stored" now that I've thought about it _way too long_. But you're right, we do start with "rotation and retention."

File: modules/microshift-audit-logs-config-intro.adoc (hunk 7 above)
Comment (ShaunaDiaz) [reply modules/microshift-audit-logs-config-intro.adoc:31]: Yes, we reference those here: https://76743--ocpdocs-pr.netlify.app/microshift/latest/microshift_configuring/microshift-audit-logs-config.html#about-audit-log-profiles_microshift-audit-logs-config. It's before the config module in the assembly.

**Pull request merged**